lib/models/builder.js

```diff
@@ -64,18 +72,31 @@ class Builder extends CoreObject {
       options = {
         tmpdir: tmpDir,
       };
-    } else {
-      broccoli = require('broccoli-builder');
-      if (this.systemTemp) {
-        console.warn('EMBER_CLI_SYSTEM_TEMP only works in combination with EMBER_CLI_BROCCOLI_2');
+
+      try {
+        this.builder = new broccoli.Builder(this.tree, options);
+        return;
+      } catch (e) {
+        // Catch here to trap InvalidNodeError. If this is thrown, it's because the node provided is not valid
+        // and likely uses the old .read/.rebuild API, so fallback to broccoli-builder that supports that API
+        if (!e instanceof broccoli.Builder.InvalidNodeError) {
+          throw e;
+        }
+
+        // Fallback to broccoli-builder
+        console.log('');
+        console.warn(chalk.red('ERROR: Invalid Broccoli2 node detected, falling back to broccoli-builder. Broccoli error:'));
+        console.warn(chalk.yellow('---------------'));
+        console.warn(chalk.yellow(e.message));
+        console.warn(chalk.yellow('---------------'));
+        this.broccoli2 = false;
```

Comment:
I just removed these properties in beta branch, I'll merge upwards so that shows up on master in a few minutes